_protected/app/configs/bans/burner-email-providers-license.txt

```diff
@@ -0,0 +1,7 @@
+Copyright 2018 Wes Bos
```

Comment:
Is this really useful?, since you mentioned the source at https://github.com/pH7Software/pH7-Social-Dating-CMS/pull/583/files#diff-5f93f02321632065cc12f46505f605dfR3

Reply:
The MIT license requires that the actual license is distributed along the software. So I'd say yes, but my knowledge in such legal issues is limited tbh.

Reply:
Then, let's leave it :)